entries/s/similarweb.com.json

```
@@ -0,0 +1,13 @@
+{
+	"Similarweb": {
+		"domain": "similarweb.com",
+		"tfa": [
+			"email"
+		],
+		"documentation": "https://support.similarweb.com/hc/en-us/articles/360016545238-Two-Step-Verification",
+		"keywords": [
+			"marketing",
+			"domains"
```

The `domains` keyword is used for registrars and resellers.
```suggestion
			"marketing"
```